ui/lib/core/addon/components/confirmation-modal.js

```diff
@@ -5,7 +5,8 @@
 import Component from '@glimmer/component';
 /**
  * @module ConfirmationModal
- * ConfirmationModal components wrap the <Hds::Modal> component to present a critical (red) type-to-confirm modal.
+ * ConfirmationModal components wrap the <Hds::Modal> component to present a critical (red) type-to-confirm modal
```

Comment:
can we update this file to be called `confirm-modal` to match the `confirm-modal.hbs` file for consistency? :)

Reply:
There are two components - I know it's confusing! But they have slightly different functionalities. As a future improvement, hopefully we can combine them..but it was outside the scope of this PR